tvdb_v4_official.py

```diff
@@ -337,12 +333,12 @@ def get_updates(self, since: int) -> list:
 
     def get_all_tag_options(self, page=0) -> list:
         """Returns a list of tag options"""
-        url = self.url.tag_options_url(page)
+        url = self.url.tag_option_url(page)
```

Comment:
This shouldn't change the url function name. Instead, the tag_options_url() function needs to accept a page arg. If you want to do something with the page arg, the url assignment in that function could also be changed, as in the second line of this suggested function:

```
   def tag_options_url(self, page=0):
        url = "{}/tags/options?page={}".format(self.base_url, page)
```